kernel/src/main/java/org/kframework/kprovex/KProve.java

```diff
@@ -85,6 +87,18 @@ public int run() {
             }
         }
 
+        if (kproveOptions.emitJsonSpec != null) {
+            try {
+                String out = new String(ToJson.apply(specModule));
+                PrintWriter fOut = new PrintWriter(kproveOptions.emitJsonSpec);
+                fOut.println(out);
+                fOut.close();
+            } catch (FileNotFoundException e) {
+                System.err.println("Could not open node output file: " + kproveOptions.emitJsonSpec);
+                e.printStackTrace();
+            }
```

Comment:
```suggestion
            FileUtil.save(new File(kproveOptions.emitJsonSpec), ToJson.apply(specModule));
```

Reply:
I should also fix the location in the source-code that I copy-pasted this pattern from.

Reply:
In `StateLog.java` I believe.

Reply:
Fixed.